server/setup.go

```diff
@@ -57,6 +57,11 @@ type factoryConfig struct {
 }
 
 func getDBFactory(ctx context.Context, cfg factoryConfig) (persistence.DBFactory, error) {
+	// I ripped out mysql to make my life simpler for now. need to find out how to stand up RDS
+	// and talk to it
+	log.Println("Creating in-memory factory")
+	return memory.NewFactory(), nil
```

Comment:
🔍  Why not achieve this with `-db=memory` in the cmd line args on startup?

Reply:
It doesn't appear easy to pass args into the docker container; can this be set with an environment var? It appears those are much easier to set up

Reply:
We should update the Dockerfile to use ARGS so that we can capitalize on this.

Reply:
Link to documentation? Anything I can find talks about `ARG`, which appears to be a build-time arg, not a runtime arg